cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/healing/HealingIntervalEntity.java

```diff
@@ -0,0 +1,98 @@
+import com.exactpro.cradle.healing.HealingInterval;
+import com.exactpro.cradle.healing.RecoveryState;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.time.LocalTime;
+import java.util.UUID;
+
+import static com.exactpro.cradle.cassandra.StorageConstants.HEALING_INTERVAL_END_TIME;
+import static com.exactpro.cradle.cassandra.StorageConstants.HEALING_INTERVAL_ID;
+import static com.exactpro.cradle.cassandra.StorageConstants.HEALING_INTERVAL_START_TIME;
+import static com.exactpro.cradle.cassandra.StorageConstants.INSTANCE_ID;
+import static com.exactpro.cradle.cassandra.StorageConstants.RECOVERY_STATE_JSON;
+
+@Entity
+public class HealingIntervalEntity
+{
+    private static final Logger logger = LoggerFactory.getLogger(HealingIntervalEntity.class);
+
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    @PartitionKey(0)
+    @CqlName(INSTANCE_ID)
+    private UUID instanceId;
+
+    @PartitionKey(1)
+    @CqlName(HEALING_INTERVAL_ID)
+    private String healingIntervalId;
+
+    @CqlName(HEALING_INTERVAL_START_TIME)
+    private LocalTime startTime;
+
+    @CqlName(HEALING_INTERVAL_END_TIME)
+    private LocalTime endTime;
+
+    @CqlName(RECOVERY_STATE_JSON)
+    private String recoveryStateJson;
+
+    public HealingIntervalEntity()
+    {
+    }
+
+    public HealingIntervalEntity(HealingInterval interval, UUID instanceId)
+    {
+        this.healingIntervalId = interval.getId();
+        this.startTime = interval.getStartTime();
+        this.endTime = interval.getEndTime();
+
+        try {
+            this.recoveryStateJson = mapper.writeValueAsString(interval.getRecoveryState());
+        } catch (JsonProcessingException e) {
+            logger.error("Error with converting recovery state to JSON format", e);
+        }
+
+        this.instanceId = instanceId;
+    }
+
+    public UUID getInstanceId()
+    {
+        return instanceId;
+    }
+
+    public void setInstanceId(UUID instanceId)
+    {
+        this.instanceId = instanceId;
+    }
+
+    public String getHealingIntervalId() { return healingIntervalId; }
+
+    public void setHealingIntervalId(String healingIntervalId) { this.healingIntervalId = healingIntervalId; }
+
+    public LocalTime getStartTime() { return startTime; }
+
+    public void setStartTime(LocalTime startTime) { this.startTime = startTime; }
+
+    public LocalTime getEndTime() { return endTime; }
+
+    public void setEndTime(LocalTime endTime) { this.endTime = endTime; }
+
+    public String getRecoveryStateJson() { return recoveryStateJson; }
+
+    public void setRecoveryStateJson(String recoveryStateJson) { this.recoveryStateJson = recoveryStateJson; }
+
+    public HealingInterval asHealingInterval() throws IOException {
+        logger.debug("RECOVERY STATE Json: " + recoveryStateJson);
```

Comment:
Debug output (if it is needed here) should use toString() to print info as one text. 
In current implementation output from other classes may appear between calls to logger.debug.